conf/modules.config

```diff
@@ -167,19 +155,13 @@ process {
         ext.args   = '--record-count 1000000 --seed 1'
         ext.prefix = { "${meta.id}.subsampled" }
         publishDir = [
-            path: { "${params.outdir}/sample_fastq/fastq" },
-            mode: params.publish_dir_mode,
```

Comment:
Mode looks accidentally deleted here.

Reply:
@maxulysse I'll smuggle a fix into one of the other PRs

Reply:
@maxulysse will fix in other PRs- thanks @mahesh-panchal !